apps/connect/release_notes.md

```diff
@@ -1,21 +1,18 @@
 # ftrack Connect release Notes
 
-## 24.5.0rc1
+
+## 24.5.0
+2024-04-03
 
 * [changed] Connect installer removed, now using a script to call the ftrack-app-installer library to package connect and codesign it on all platforms.
 * [changed] New versioning system matching all backlight products.
-
-## v3.1.0rc1
-2024-04-22
-
 * [changed] Allow non versioned connect plugins, supporting legacy actions naming convention.
 * [changed] Fixed overlay widget bugs.
 * [new] PySide6 support; still not fully compatible until riffle is removed.
```

Comment:
Update this now that riffle is supported? And the PySide2 support below is not needed right, as it had PySide2 before support before this release?